src/hotplug_libudev.c

```diff
@@ -75,7 +75,7 @@ extern char Add_Serial_In_Name;
 static pthread_t usbNotifyThread;
 static int driverSize = -1;
 static struct udev *Udev;
-
+static char * hpDirPath = PCSCLITE_HP_DROPDIR;
```

Comment:
Why set a value for `hpDirPath` here?
The value will be set at lines 118-122. No?